crates/driver/Cargo.toml

```diff
@@ -60,6 +60,7 @@ tracing = { workspace = true }
 warp = { workspace = true }
 
 [dev-dependencies]
+approx = { workspace = true }
```

Comment:
There's no reason to share this dependency. The only reason you really need to fix a dependency version across crates is if you are using it in the API of the crate and need consistency. I would say otherwise let the creates pick their own versions, especially considering that we've already been introducing this autonomy to the `driver` and `solvers` in hopes of one day separating them out into their own repos.

Reply:
I forgot we were intentionally not sharing dependencies. There is a reason though, which is lower compile times by not accidentally using multiple versions of the same crate, which includes different feature sets too. Anyway, will change it to normal dep.